lib/src/server/server.dart

```diff
@@ -99,7 +102,7 @@ class Server {
       server = _secureServer;
```

Comment:
You need to also pass the parameters in the secure case

Reply:
done